### ISSUE TITLE
about loss functions

### ISSUE THREAD

**Issue by Tihetna1891** [Hzzone/DU-GAN#7]
Hello, could you help explain how the results of the loss function are saved and how graphs may be extracted from them?

**Comment by Hzzone** [comment]
I do not understand what you mean.

**Comment by Tihetna1891** [comment]
So, when I train other models like WGAN, the loss values are saved, but I couldn't find any in this one...and I want the numbers to build a graph like this one. 
![image](https://user-images.githubusercontent.com/102287467/171987152-af620ab1-a8eb-4b06-8496-714db58a6c46.png)

**Comment by Hzzone** [comment]
The code prints the logs, without saving in tensorboard and etc, see https://github.com/Hzzone/DU-GAN/blob/308d6d0d858559c29282de915a3945d2240627e6/utils/loggerx.py#L78

You may add a new line after L78, using tensorboard https://pytorch.org/docs/stable/tensorboard.html?highlight=tensorboard

**Comment by Tihetna1891** [comment]
thanks I will try it

**Comment by Tihetna1891** [comment]
Hello sir , I tried to save the logs using tensorboard as u have said but
the loss graph did not appear in the tensorboars ..if that is not much
truble could you please send me the suitable code that will be add after L78
